pkg/job/v1alpha3/job/job.go

```diff
@@ -28,9 +28,10 @@ func (j Job) GetDeployedJobStatus(
 			log.Error(unerr, "NestedFieldCopy unstructured to status error")
 			return nil, unerr
 		}
-		log.Info("NestedFieldCopy unstructured to status error",
-			"err", "Status is not found in job")
-		return &jobCondition, nil
+		// Job does not have the running condition in status, thus we think
+		// the job is running when it is created.
+		log.Info("NestedFieldCopy", "err", "status cannot be found in job")
```

Comment:
Also, why is this logic only added for batch job and not for other types?

Reply:
I meant, kubeflow provider

Reply:
The kubeflow provider already implements it before.

Reply:
https://github.com/kubeflow/katib/blob/master/pkg/job/v1alpha3/kubeflow/kubeflow.go#L39